python/ray/serve/master.py

```diff
@@ -197,13 +180,15 @@ def _get_or_start_http_proxy(self, node_id, host, port):
             ).remote(
                 host, port, instance_name=self.instance_name)
 
+        self.router = self.http_proxy
```

Comment:
Change name?

Reply:
Read the comment, fair enough. If we continue down the path of them being merged we should probably remove this alias for clarify.